### ISSUE TITLE
[DigitalHive] error triggered while searching

### ISSUE THREAD

**Issue by kormic911** [Jackett/Jackett#1266]
Following error received when attempting to search DigitalHive using keyword 'badlands'

```
04-10 10:56:33 Error An error occured during manual search on DigitalHive:  One or more errors occurred. System.AggregateException: One or more errors occurred. ---> System.FormatException: The string was not recognized as a valid DateTime. There is an unknown word starting at index 19.
  at Jackett.Indexers.BaseIndexer.OnParseError (System.String results, System.Exception ex) <0x412b61f0 + 0x00143> in <filename unknown>:0 
  at Jackett.Indexers.DigitalHive+<PerformQuery>c__async2.MoveNext () <0x412445b0 + 0x00a83> in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Threading.Tasks.Task.ThrowIfExceptional (Boolean includeTaskCanceledExceptions) <0x7f38bd49daf0 + 0x00037> in <filename unknown>:0 
  at System.Threading.Tasks.Task`1[TResult].GetResultCore (Boolean waitCompletionNotification) <0x7f38bd48cb10 + 0x00066> in <filename unknown>:0 
  at System.Threading.Tasks.Task`1[TResult].get_Result () <0x7f38bd48cab0 + 0x00024> in <filename unknown>:0 
  at Jackett.Controllers.AdminController+<Search>c__AnonStorey8.<>m__2 (IIndexer indexer) <0x41286c20 + 0x000d2> in <filename unknown>:0 
---> (Inner Exception #0) System.FormatException: The string was not recognized as a valid DateTime. There is an unknown word starting at index 19.
  at Jackett.Indexers.BaseIndexer.OnParseError (System.String results, System.Exception ex) <0x412b61f0 + 0x00143> in <filename unknown>:0 
  at Jackett.Indexers.DigitalHive+<PerformQuery>c__async2.MoveNext () <0x412445b0 + 0x00a83> in <filename unknown>:0 <---
```

Jackett version:  v0.7.1272
Mono version (if not using Windows):  4.8.0 (Stable 4.8.0.524/9d74414 Wed Apr  5 17:57:04 UTC 2017)

-------------------------------

**Comment by kormic911** [comment]
After digging a bit deeper this is what a normal string it expects looks like:
`<br><span style='font-size: 12px !important;'>Added: 2016-10-28 14:22:02&nbsp;<td class='thicon'><a href="download.php?id=xxx&https=no&name=xxx.torrent"><i class='fa fa-download'></i></a></td>`

But once you download a file it changes to something like this
`<br><span style='font-size: 12px !important;'>Added: 2016-01-15 04:57:37<br><a href='snatchinfo.php'>Seed Time: <font style="color:#FF0000;">N/A</font></a> <a href='snatchinfo.php'>Ratio: <font style="color:#FF0000;">N/A</font></a><td class='thicon'><a href="download.php?id=xxx&https=no&name=xxx.torrent"><i class='fa fa-download'></i></a></td>`

**Comment by flightlevel** [comment]
Had a go, see if the latest release fixes it, very hard without access

**Comment by kormic911** [comment]
That doesn't seem to work, I think something along the lines of this would probably work better:
`qRow.Find("td:nth-child(2) > span").First().Text().ReadLine().Trim().Replace("Added: ", "").Split('<br>')[0];`

**Comment by kaso17** [comment]
Should be fixed with 2e7f2056c4cab2f326a0f92ae4216fa546ef6b8d
